GettingStarted.md

```diff
@@ -134,25 +134,54 @@ You can run Kontain payload wrapped in a native Docker container.... in this cas
 **** KRUN is not in the bundle yet, skip this section ****
 
 Or you can use Kontain `krun` runtime from docker/podman or directly. `krun` is installed together with KM and other components.
```

Comment:
1. Would be nice to add "For an intro to runtime, see misc. blogs, e.g. https://medium.com/@avijitsarkar123/docker-and-oci-runtimes-a9c23a5646d6"
2. overall looks good, but as I mentioned an actual example (the one I can cut-n-paste and run) is needed. I suggest using something like 
```
docker pull kontainapp/runenv-python
docker run --runtime=runk kontainapp/runenv-python -c "import os; print(os.uname())"
```
we can work offline on this